internal/cmd/backup_test.go

```diff
@@ -281,9 +281,13 @@ func TestBackupCreateCmdFunc(t *testing.T) {
 
 	ctx := t.Context()
 	srv := zedtesting.NewTestServer(ctx, t)
+	runErrs := make(chan error, 1)
 	go func() {
-		require.NoError(t, srv.Run(ctx))
+		err := srv.Run(ctx)
+		runErrs <- err
 	}()
+	err := <-runErrs
+	require.NoError(t, err)
```

Comment:
These two refactors are `testifyrequire` lint fixes that I did as drive-bys. We don't have that linter enabled yet. I'd like to do that soon.